multiversx_sdk_cli/cli_deps.py

```diff
@@ -16,7 +16,6 @@ def setup_parser(subparsers: Any) -> Any:
     sub = cli_shared.add_command_subparser(subparsers, "deps", "install", "Install dependencies or multiversx-sdk modules.")
     sub.add_argument("name", choices=choices, help="the dependency to install")
     sub.add_argument("--overwrite", action="store_true", default=False, help="whether to overwrite an existing installation")
-    sub.add_argument("--tag", help="the tag or version to install")
```

Comment:
why remove the tag? So we can make sure that compatible dependencies are used? Can you mark the flag as deprecated instead of removing it?

Reply:
yes, we want to make sure that the dependencies are compatible with each other. we plan to remove it from all commands and as it was hardly ever used we will remove it in the next release.